### PR TITLE
javascript-primitives-and-variables -Zach Giffen

### PR DESCRIPTION
**Pull request by Zach-Giffen**
java stuff

### CODE REVIEW

File: javascript-primitives-and-variables/main.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const fullName = 'Zach Giffen';
+const isCool = true;
+const totalPets = 4;
+
+console.log(fullName);
```
Comment (robgmerrill) [inline javascript-primitives-and-variables/main.js:5]: According to step 9 these should all have labels:
**9. Log each of your variables to the console with labels.**

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great start! These should all have labels according to step 9.

File: javascript-primitives-and-variables/main.js (hunk 1 above)
Comment (robgmerrill) [reply javascript-primitives-and-variables/main.js:5]: eg. `console.log("fullName", fullName);

Comment (Zach-Giffen) [reply javascript-primitives-and-variables/main.js]: java stuff 4 real

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**